src/amo/middleware/responseTime.js

```diff
@@ -26,14 +26,14 @@ export const responseTime = ({
       // TODO: generate a key based on the rendered component, which I think is
       // in server/base.js -> match() -> renderProps.components
 
-      client.increment(`response_code.${res.statusCode}.count`);
+      client.increment(`response_status.${res.statusCode}.count`);
```

Comment:
if we're changing this, my suggestion would be to change it to match the format addons-server (appears to, from looking in grafana), uses - `response.${res.statusCode}`

Reply:
(Or if this is a better format, log a follow-up to adjust the addons-server statsd config.)

Reply:
Changing the keys on addons-server would be problematic for 2 reasons: (1) it would make the existing data/dashboards out of sync, and (2) it's probably hidden in a django plugin.

That being said, these keys are much clearer and I don't want to mix status codes and methods, which is why the prefix is different and a bit longer than `response`.

Reply:
After a quick look at Grafana, we do not have metrics for methods alone, it's always per view, so I guess we can change `response_status` to `response` and maybe drop the `.count` although that is kinda useful too, but we would have to keep `response_method`.